app/code/community/Ebizmarts/MailChimp/sql/mailchimp_setup/mysql4-upgrade-1.1.12-1.1.13.php

```diff
@@ -60,7 +60,6 @@
 ;
 $attribute->save();
 
-Mage::helper('mailchimp')->saveMailChimpConfig(array(array(Ebizmarts_MailChimp_Model_Config::ENABLE_POPUP, 0)), 0, 'default');
-
+$installer->deleteConfigData(Ebizmarts_MailChimp_Model_Config::ENABLE_POPUP);
```

Comment:
are we sure `deleteConfigData` performs the same on all Magento 1 versions we support?